lib/tasks/fix_missing_organisations.rake

```diff
@@ -0,0 +1,20 @@
+#rake goodcity:fix_missing_organisations
+
+namespace :goodcity do
+  task fix_missing_organisations: :environment do
+    count = 0
+    log = Goodcity::RakeLogger.new("fix_missing_organisations")
+    #get all the gc orders without organisation and not draft draft orders
+    Order.where("orders.state <> ? and orders.organisation_id IS NULL and orders.created_by_id IS NOT NULL and code ILIKE ?","draft", "GC-%").find_each do |order|
+      creator_organisations = order.created_by&.organisations
+      next unless creator_organisations
```

Comment:
What if you get blank active relation like : `#<ActiveRecord::Associations::CollectionProxy []>` Will next get executed?

Reply:
I think next will not get executed, will add a `blank?` check here, so that we make sure it doesn't break.